zadani/soft/style.css

```diff
@@ -1,43 +1,304 @@
+                                                                                                                
+                                                                                                                .perex {
+                                                                                                                    padding: 0px 20px 0px 20px;
+                                                                                                                    margin: 0;
+                                                                                                                }
+                                                                                                                
+                                                                                                                .autor {
+                                                                                                                    padding-top: 20px;
+                                                                                                                    font-style: italic;
+                                                                                                                    position: relative;
+                                                                                                                }
+                                                                                                                
+                                                                                                                .autor::before {
+                                                                                                                    content: '';
+                                                                                                                    width: 15%;
+                                                                                                                    border-bottom: solid 2px rgba(255, 255, 255, .5);
+                                                                                                                    position: absolute;
+                                                                                                                    left: 44%;
+                                                                                                                    top: 30%;
+                                                                                                                    /*jak zajistit aby bylo podtržení v každé verzi (mobil desktop tablet) uprostřed stránky? zkoušela jsem přes procenta ale nepodařilo se - napadlo mě nastavit to pro mobil desktop i tablet zvlášť pro určitou min-width, ale pak by se podtržení asi stejně posouvalo při každém zvětšení rozlišení*/
+                                                                                                                    z-index: 1;
```

Comment:
Je to proto, že využíváš relativní jednotky a odsazuješ o 44% od kraje prvku, takže spoléháš na to, že bude nějak široký, aby to vždy vyšlo do půlky.
Bylo by lepší dát mu pevnou šířku v px a mohli bychom ho vycentrovat buď pomocí CSS kalkulačky (CSS calc) https://developer.mozilla.org/en-US/docs/Web/CSS/calc() - to ještě neznáme.
Nebo využít transformací a dát mu left: 50%; a transform: translate(-50%, 0); - to by ho mělo vystředit vždy.